deploy/.gitignore

```
@@ -0,0 +1 @@
+/.gotools/
```

do we need first `/`?
```suggestion
.gotools/
```

---

Leading `/` was intentional, means "just in this dir". But thinking about it, it makes sense to remove ignore it everywhere...

---

Moved to root.